Content.Client/_DV/CartridgeLoader/Cartridges/NanoChatLookupView.xaml.cs

```diff
@@ -0,0 +1,78 @@
+using System.Numerics;
+using Content.Shared._DV.CartridgeLoader.Cartridges;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client._DV.CartridgeLoader.Cartridges;
+
+[GenerateTypedNameReferences]
+public sealed partial class NanoChatLookupView : PanelContainer
+{
+    public NanoChatLookupView()
+    {
+        RobustXamlLoader.Load(this);
+    }
+
+    public event Action<NanoChatRecipient>? OnStartChat;
+
+    public void UpdateContactList(NanoChatUiState state)
+    {
+        ContactsList.RemoveAllChildren();
+        if (state.Contacts is not { } contacts)
+        {
+            ContactsList.AddChild(new Label() { Text = Loc.GetString("nano-chat-look-up-no-server") });
+            return;
+        }
+
+        for (var idx = 0; idx < contacts.Count; idx++)
+        {
+            var contact = contacts[idx];
+            var isEvenRow = idx % 2 == 0;
+            var contactControl = new ContactControl(contact, state, isEvenRow, OnStartChat);
+            ContactsList.AddChild(contactControl);
+        }
+    }
+    public sealed class  ContactControl : Control
```

Comment:
This should be PanelContainer instead of Control for the PanelBackground style classes to work

Reply:
Also, superfluous space